test/test_exporter.py

```diff
@@ -354,11 +317,12 @@ def test_sum_delta(self, mock_post):
 
     @patch.object(requests.Session, 'post')
     def test_sum_delta_non_monotonic(self, mock_post):
```

Comment:
I think this is not correct, but this also falls into the area of a potential follow up

Reply:
Yes, this was not correct. As of [2d01775](https://github.com/dynatrace-oss/opentelemetry-metric-python/pull/27/commits/2d01775f272252daa002b671d1a760238bbcfc8c), non monotonic delta sums are now dropped and test are updated accordingly. Thanks for catching this.